icekit/articles/abstract_models.py

```diff
@@ -1,12 +1,12 @@
-from icekit.abstract_models import FluentFieldsMixin
+from urlparse import urljoin
+
+from django.template.response import TemplateResponse
+
 from icekit.publishing.models import PublishingModel
-from django.db import models
+from polymorphic.models import PolymorphicModel
```

Comment:
Of course, if we know about any minimum version requirements they should be specified in `setup.py`. Same with any maximum version requirements or known incompatible versions (e.g. containing bugs that were subsequently fixed) to be excluded. However, until we actually know about a feature or implementation that is broken or missing, we don't need to add a random minimum version spec to every requirement. In this case, we know that imports were changed in 0.8 and if we don't want to support pre/post 0.8 with a try/except, we should exclude the known incompatible versions.

If it's easy enough (and a moved import seems pretty easy), I vote for maintaining compatibility. I believe elsewhere in ICEkit code we do try/except this import. But given the number of ICEkit installations out there and the low probability of needing to upgrade any of them to latest ICEkit without also being able to upgrade polymorphic, I'm also fine with dropping support for `<=0.8`. In which case we can remove the other try/except imports (if there are any).